src/gui/search/searchwidget.cpp

```diff
@@ -162,6 +163,14 @@ bool SearchWidget::eventFilter(QObject *object, QEvent *event)
             closeTab(tabIndex);
             return true;
         }
+        else if (mouseEvent->button() == Qt::RightButton)
+        {
+            QMenu *menu = new QMenu(this);
```

Comment:
Memory leak, you need to add `menu->setAttribute(Qt::WA_DeleteOnClose);`

Reply:
thank you!